NOnion/Utility/FSharpUtil.fs

```diff
@@ -65,7 +65,7 @@ module FSharpUtil =
                 return failwith "unreachable"
         }
 
-    let Retry<'E1, 'E2 when 'E1 :> Exception and 'E2 :> Exception>
+    let Retry<'E1 when 'E1 :> Exception>
```

Comment:
@webwarrior-ws if there's only 1 exception type parameter now, there's no point in naming it E1, please rename it to TEx